alina.css

```
@@ -0,0 +1,200 @@
+body {
+    margin: 0;
+}
+.block {
+    height: 100vh;
+    width: 100%;
+    position: relative;
+}
+
+/* nav bar */
+.nav {
+    height: 100vh;
+    width: 6%;
+    position: fixed;
+    z-index: 2; 
+    background-color:transparent;
+    overflow-x: hidden; 
+    padding-top: 40vh;
+}
+.nav a {
+    display: block;
+    margin-left: 10%;
+    padding-block-start: 3vh;
+}
+.icon {
+    max-height: 8vh;
+    max-width: 100%;
+}
+
+/* text properties */
+.heading-light {
+    font-family:cursive;
+    background-color: transparent;
+    color:#f7ede2;
+    font-size: 9vh;
+    white-space: nowrap;
+    overflow:hidden;
+    margin: 0;
+}
+.heading-dark {
+    font-family: cursive;
+    background-color: transparent;
+    color: #000000;
+    font-size: 9vh;
+    white-space: nowrap;
+    overflow:hidden;
+    margin: 0;
+}
```

There is some repetition here. It might be possible to consolidate similar properties into a single class, and then add the property that is different in another class or ID.

```suggestion
.heading {
    font-family:cursive;
    background-color: transparent;
    font-size: 9vh;
    white-space: nowrap;
    overflow:hidden;
    margin: 0;
}

#dark-header {
    color: #000000;
}
    
#light-header {
    color:#f7ede2;;
}
```